static/src/javascripts/projects/commercial/modules/consentless/init-fixed-slots.ts

```diff
@@ -10,11 +10,10 @@ const initFixedSlots = (): Promise<void> => {
 
 	// define slots
 	adverts.forEach((slotElement) => {
-		const slotName = slotElement.dataset.name?.includes('inline')
-			? 'inline'
-			: slotElement.dataset.name;
+		const slotName = slotElement.dataset.name;
+		const slotKind = slotName?.includes('inline') ? 'inline' : undefined;
 		if (slotName) {
-			defineSlot(slotElement, slotName);
+			defineSlot(slotElement, slotName, slotKind);
```

Comment:
Although a bit more verbose, I think separate `defineFixedSlot `and `defineInlineSlot` or similar might be better?

Reply:
I really like this idea. I'm going to hold off on this PR as I think the interface for `defineSlot` is going to change once when we change the slot names, and we can implement something along these lines then.